deployment/keystone/types.go

```diff
@@ -229,7 +229,51 @@ type RegisteredDon struct {
 	Nodes []deployment.Node
 }
 
-func (d RegisteredDon) signers(chainFamily string) []common.Address {
+type RegisteredDonConfig struct {
+	Name             string
+	NodeIDs          []string // ids in the offchain client
+	RegistryChainSel uint64
+}
+
+func NewRegisteredDon(env deployment.Environment, cfg RegisteredDonConfig) (*RegisteredDon, error) {
```

Comment:
so we can create the needed input to ConfigureForwarderContracts for a given env